src/ArrayValue.ts

```diff
@@ -6,7 +6,7 @@
 import {ArraySize} from './ArraySize'
 import {CellError} from './Cell'
 import {EmptyValue, InternalScalarValue, InterpreterValue} from './interpreter/InterpreterValue'
-import {SimpleRangeValue} from './interpreter/SimpleRangeValue'
+import {SimpleRangeValue} from './SimpleRangeValue'
```

Comment:
Typedoc is configured in such a way that is doesn't generate documentation for the code inside the `src/interpreter` directory. We should update typedoc to the newest version and reconfigure it properly. This will be done in #1086 

As a temporary solution, I moved `SimpleRangeValue` class from `src/interpreter` to `src`

Reply:
Have you tried rexporting `./interpreter/SimpleRangeValue` from `./index`? That would be cleaner.

Reply:
As it is used across the engine, not just inside interpreter, then I suppose to previous location of this file was wrong anyway.

Reply:
> Looking at history, probably you made the change because reexporting did not work.

It was exported by `index.ts` even before this PR, so our clients were able to use it but didn't know how to use it (it weren't present in the documentation).